src/core/src/op/constant.cpp

```diff
@@ -93,7 +93,33 @@ Constant::Constant(const Tensor& tensor)
 }
 
 Constant::Constant(const element::Type& type, const Shape& shape, const std::vector<std::string>& values)
-    : Constant(type, shape, from_string_vector(values)) {
+    : Constant(false, type, shape) {
+    // TODO: Rework code as this is a modified duplicate of another templated ctor
```

Comment:
can we apply this TODO?

Reply:
Removed TODO, the current duplication is not big. We may think of how to avoid in future.
Propose to leave it as is.